dcrsqlite/apisource.go

```diff
@@ -468,3 +468,104 @@ func (db *wiredDB) GetMempoolSSTxDetails(N int) *apitypes.MempoolTicketDetails {
 	}
 	return &mpTicketDetails
 }
+
+//GetAddressTransactions returns an apitypes.Address Object
+//with at most the last 10 transactions the address was in
+func (db *wiredDB) GetAddressTransactions(addr string, count int) *apitypes.Address {
+	address, err := dcrutil.DecodeAddress(addr)
+	if err != nil {
+		log.Errorf("Invalid address %s", addr)
+		return nil
+	}
+	txs, err := db.client.SearchRawTransactionsVerbose(address, 0, count, true, false, make([]string, 0))
+	if err != nil {
+		log.Errorf("GetAddressTransactions failed for address %s", addr)
+		return nil
+	}
+	tx := make([]*apitypes.AddressTxShort, 0)
+	for i := range txs {
+		txarray := make([]apitypes.AddressTxValue, 0)
+		for j := range txs[i].Vin {
+			for c := range txs[i].Vin[j].PrevOut.Addresses {
+				if txs[i].Vin[j].PrevOut.Addresses[c] == addr {
+					txarray = append(txarray, apitypes.AddressTxValue{
+						ValueType: "Vin",
+						Amount:    txs[i].Vin[j].PrevOut.Value,
+					})
+				}
+			}
+		}
+		for j := range txs[i].Vout {
+			for c := range txs[i].Vout[j].ScriptPubKey.Addresses {
+				if txs[i].Vout[j].ScriptPubKey.Addresses[c] == addr {
+					txarray = append(txarray, apitypes.AddressTxValue{
+						ValueType: "Vout",
+						Amount:    txs[i].Vout[j].Value,
+					})
+				}
+			}
+		}
+		tx = append(tx, &apitypes.AddressTxShort{
+			TxID:   txs[i].Txid,
+			Time:   txs[i].Time,
+			Values: txarray,
+		})
+	}
+	return &apitypes.Address{
+		Address:      addr,
+		Transactions: tx,
+	}
+
+}
+
+//GetAddressTransactions returns an array of apitypes.AddressTxRaw
+//objects representing the raw result of SearchRawTransactionsverbose
+func (db *wiredDB) GetAddressTransactionsRaw(addr string, count int) []*apitypes.AddressTxRaw {
+	address, err := dcrutil.DecodeAddress(addr)
```

Comment:
If you wipe out vendor/ and `glide install` from your branch, you are using older dependencies.  While a merge of the PR would handle this, it's best for you to be working on a branch that is really on top of master.  Also, when switching branches, it's best to always `rm -rf vendor` and `glide install`.

I saw a rebase master gave you some trouble, so we can go through this interactively if you want.  Resolving conflicts can be a pain without the right tools and expectations for what git is going to say at certain points.